validate_env.py

```
@@ -1,74 +1,102 @@
+        )
 
-def check_anthropic_connectivity():
-    """Verifies the Anthropic API (The New Brain) is reachable."""
-    api_key = os.getenv("ANTHROPIC_API_KEY")
+        response = client.chat.completions.create(
+            model="glm-4.7",
+            messages=[{"role": "user", "content": "test"}],
+            max_tokens=10
+        )
+
+        return True, f"Z.ai Connected (model: {response.model})"
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)